src/Microsoft.Azure.WebJobs/WebJobs.csproj

```diff
@@ -25,6 +25,7 @@
     </PackageReference>
     <PackageReference Include="System.ComponentModel.Annotations" Version="4.4.0" />
     <PackageReference Include="System.Diagnostics.TraceSource" Version="4.3.0" />
+    <PackageReference Include="System.Memory.Data" Version="1.0.1" />
```

Comment:
Does this PR changes uses any type from this package?  I couldn't spot any. Remove if not used.

Reply:
Yup, BinaryData. WebJobs is on the version of .net standard that does not have BinaryData built-in